en/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: news_post
+title: "Ruby 2.4.0-preview2 Released"
+author: "naruse"
+translator:
+date: 2016-09-08 18:00:00 +0900
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.4.0-preview2.
+
+Ruby 2.4.0-preview2 is the second preview of Ruby 2.4.0.
+This preview2 is released to get feedback from community.
+Feel free to
+[send feedback](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)
+since you can still change the features.
+
+## [Unify Fixnum and Bignum into Integer](https://bugs.ruby-lang.org/issues/12005)
+
+Though [ISO/IEC 30170:2012](http://www.iso.org/iso/iso_catalogue/catalogue_tc/catalogue_detail.htm?csnumber=59579)
+doesn't specify details of the Integer class,
+CRuby had two visible Integer classes: Fixnum and Bignum.
+Ruby 2.4 unifies them into Integer.
+All C extensions which touches Fixnum or Bignum class needs to fix.
+
+See also [the ticket](https://bugs.ruby-lang.org/issues/12005) and [akr's slide](http://www.a-k-r.org/pub/2016-09-08-rubykaigi-unified-integer.pdf).
+
+## [String supports Unicode case mappings](https://bugs.ruby-lang.org/issues/10085)
+
+`String/Symbol#upcase/downcase/swapcase/capitalize(!)` now handle
+Unicode case mappings instead of only ASCII case mappings.
+
+## Performance improvements
+
+Ruby 2.4 also contains the following performance improvements including
+language changes:
+
+### [Array#max, Array#min](https://bugs.ruby-lang.org/issues/12172)
+
+`[x, y].max` and `[x, y].min` are optimized to not create a temporary array
+under certain conditions.
+
+### [Regexp#match?](https://bugs.ruby-lang.org/issues/8110)
+
+Added `Regexp#match?`, which executes a regexp match without creating
+a back reference object and changing `$~` to reduce object allocation.
+
+### Other performance improvements
+
+* [speed up instance variable access](https://bugs.ruby-lang.org/issues/12274)
+
+## Debugging
+
+### [Thread#report_on_exception and Thread.report_on_exception](https://bugs.ruby-lang.org/issues/6647)
+
+Ruby ignores exceptions in threads unless another thread explicitly joins them.
+With `report_on_exception = true`,
+you can notice if a thread has died due to an unhandled exception.
+
+Send us feedback what should be the default for `report_on_exception`
+and about report-on-GC.
+
+### [Thread deadlock detection now shows threads with their backtrace and dependency](https://bugs.ruby-lang.org/issues/8214)
+
+Ruby has deadlock detection around waiting threads, but its report doesn't
+include enough information for debugging.
+Ruby 2.4's deadlock detection shows threads with their backtrace and
+dependent threads.
+
+Try and enjoy programming with Ruby 2.4.0-preview2, and
+[send us feedback](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)!
+
+## Notable Changes since 2.3
+
+* Support OpenSSL 1.1.0
```

Comment:
Sorry, I meant will the changes that were already explained above (like Integer, ...) also appear on this list?

If not, the section title should be "**Other** Notable Changes".